nginx.conf

```diff
@@ -36,7 +34,11 @@ http {
   server {
     listen 80 default_server;
     listen [::]:80 default_server;
-
+    http2_max_field_size 64k;
+    http2_max_header_size 512k;
+    gzip on;
+    add_header 'Cache-Control' 'no-store, no-cache, must-revalidate, proxy-revalidate, max-age=0';
```

Comment:
Wouldn't this instruct the client's browser to not cache the page that nginx is sending it?